src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -25,7 +25,7 @@ object Run  {
 
   def apply(pipeline: Pipeline): Run = {
     val rpr = new RunPipelineRequest().setPipelineId(pipeline.id).setProjectId(pipeline.projectId).setServiceAccount(JesServiceAccount)
-    val tag = s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.scope.name}]"
+    val tag: String = s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.tag}]"
```

Comment:
For things which aren't part of the public api my personal stance is "Is it unclear and do I care even if it isn't?". In this case it's obviously a string.